### ISSUE TITLE
Multiple front-end cropping fields overwrite each other on save

### ISSUE THREAD

**Issue by GerritChan** [joppuyo/acf-image-aspect-ratio-crop#43]
Saving one field at a time works perfectly, multiple fields seem to overwrite each other. The attached screenshots contain the pre-save and after-save situation.

**Before saving**
![before-save](https://user-images.githubusercontent.com/25591914/101340819-87c90680-3880-11eb-9cca-3c52f6b95267.png)

**After saving**
![after-save](https://user-images.githubusercontent.com/25591914/101340815-87307000-3880-11eb-8593-1712fac4099e.png)

**Comment by joppuyo** [comment]
Hey,

Thank you for the bug report.

However, I'm unable to reproduce this issue on my end.

Could you provide me the ACF field configuration in JSON format, the PHP code you are using for rendering the form, and the steps to reproduce this issue?

Thanks!

**Comment by GerritChan** [comment]
Hello, 

No problem! 
Below you will find the code I am using, and an export of the fields in JSON. When you try to add more than one cropped image, the issue occurs. In the attached screenshots, I changed the 'hidden' input to a 'text' input, and you can see that when I crop an image in field 2, it changes the ID in the 'hidden' input from field 1. 

[acf-export-2020-12-08.zip](https://github.com/joppuyo/acf-image-aspect-ratio-crop/files/5659255/acf-export-2020-12-08.zip)
```php
acf_form(array(
      'post_id' => 'user_' . get_current_user_id(),
      'field_groups' => array('group_5fcdeb724f2b2'),
      'uploader' => 'basic',
      'submit_value'	=> 'Edit'
));
```

**Start**
![Screenshot_1](https://user-images.githubusercontent.com/25591914/101485084-ec589400-395a-11eb-876c-d716249996bd.png)

**Crop 1**
![Screenshot_2](https://user-images.githubusercontent.com/25591914/101485115-f8dcec80-395a-11eb-8afa-3b5130263b55.png)

**Crop 2**
![Screenshot_3](https://user-images.githubusercontent.com/25591914/101485123-fe3a3700-395a-11eb-838f-7dddb48b9a47.png)

**End result**
![Screenshot_4](https://user-images.githubusercontent.com/25591914/101485343-4eb19480-395b-11eb-801c-376ad9b83498.png)

**Comment by joppuyo** [comment]
Thank you!

Are these fields attached to an user? I was testing the functionality with a post which may explain why I couldn't reproduce the issue.

**Comment by GerritChan** [comment]
Yes, they are attached to a user, it's basically an 'edit profile' form.

**Comment by joppuyo** [comment]
Hey,

I think I've figured out what's causing this issue. I've found a fix but I'll have to do some more tests tomorrow before I can release a bug fix version.

**Comment by joppuyo** [comment]
I just released version v5.0.4 which contains a fix for this issue, could you try it and see if this issue has been fixed.

Thanks!

**Comment by GerritChan** [comment]
Just tested it, works like a charm! Thanks!
